build.sbt

```diff
@@ -262,6 +262,16 @@ lazy val shapelessJVM = shapeless.jvm
 lazy val shapelessJS = shapeless.js
 lazy val shapelessNative = shapeless.native
 
+lazy val scalaxml = myCrossProject("scalaxml")
+  .dependsOn(core % "compile->compile;test->test")
+  .settings(
+    libraryDependencies += "org.scala-lang.modules" %% "scala-xml" % scalaXmlVersion
```

Comment:
If scala-xml is available for Scala.js and Scala Native, shouldn't this use `%%%`?